### PR TITLE
Fix bug with smithy init on windows.

### PR DESCRIPTION
**Pull request by melund**

### CODE REVIEW

File: conda_smithy/cli.py [hunk 1]
```diff
@@ -43,7 +43,7 @@ def create_git_repo(target, msg):
         # prevent this:
         # bash: line 1: ./ci_support/run_docker_build.sh: Permission denied
         # ./ci_support/run_docker_build.sh returned exit code 126
-        subprocess.check_call(['git', 'update-index' '--chmod=+x' 'ci_support/run_docker_build.sh'], cwd=target)
+        subprocess.check_call(['git', 'update-index', '--chmod=+x', 'ci_support/run_docker_build.sh'], cwd=target)
```
Comment (ocefpaf) [inline conda_smithy/cli.py]: :scream: 
That is what you get when 99.99% of the devs don't use Windows :wink:

Comment (melund) [inline conda_smithy/cli.py]: Well. I am kind of forced :) Anyway, not as bad a situation as it was a few years ago..

**Comment by scopatz** [comment]
LGTM

**Comment by ocefpaf** [comment]
I usually let only @pelson merge this here, but that seems pretty harmless for me to merge :wink:

**Comment by pelson** [comment]
How did this even work before? String concatenation in Python means that these should be joined into an unspaced string - which shouldn't work on unix either 😕 

Anyway, thanks for the PR and thanks for merging @ocefpaf!

**Pull request merged**